stellargraph/connector/neo4j/graph.py

```diff
@@ -87,11 +87,16 @@ def to_adjacency_matrix(self, node_ids):
             nodes (list): The collection of nodes
                 comprising the subgraph. The adjacency matrix is
                 computed for this subgraph.
+            weighted (bool, optional): Must be false, at the moment.
 
         Returns:
              The weighted adjacency matrix.
         """
 
+        # this param is for compatibility with StellarGraph.to_adjacency_matrix
+        if weighted is not False:
```

Comment:
Could just be
```suggestion
        if weighted:
```

Reply:
I did the `is` form so that it had to be explicitly `False` rather than passing in a `[]` or something... but that's probably unnecessary. 🤷‍♂️